src/stellar_types.h

```diff
@@ -352,12 +560,11 @@ typedef struct {
     uint32_t rawLength;
     uint8_t hash[HASH_SIZE];
     uint16_t offset;
-    uint8_t network;
-    Operation opDetails;
-    tx_details_t txDetails;
-    uint8_t opCount;
-    uint8_t opIdx;
     uint32_t tx;
+    uint8_t network;
+    EnvelopeType envelopeType;
+    FeeBumpTransactionDetails feeBumpTxDetails;
```

Comment:
memo: putting `feeBumpTxDetails` and` txDetails` in a union can reduce memory usage